package.json

```diff
@@ -25,11 +25,12 @@
   },
   "homepage": "https://github.com/cronofy/cronofy-node#readme",
   "dependencies": {
-    "lodash": "^3.10.1",
-    "rest": "^1.3.1"
+    "lodash": "^4.17.4",
```

Comment:
Yeah, I think I now understand how NPM avoids dependency hell most the time, but when you end up in it it is more hellish than dependency managers I'm used to!